packages/make-styles/src/renderer/getStyleSheetForBucket.ts

```diff
@@ -0,0 +1,72 @@
+import { MakeStylesRenderer, StyleBucketName } from '../types';
+
+/**
+ * Ordered style buckets using their short pseudo name.
+ *
+ * @private
+ */
+export const styleBucketOrdering: StyleBucketName[] = [
+  // catch-all
+  '',
+  // link
+  'l',
+  // visited
+  'v',
+  // focus-within
+  'w',
+  // focus
+  'f',
+  // focus-visible
+  'i',
+  // hover
+  'h',
+  // active
+  'a',
+  // keyframes
+  'k',
+  // at-rules
+  't',
+];
+
+/**
+ * Lazily adds a `<style>` bucket to the `<head>`. This will ensure that the style buckets are ordered.
+ */
+export function getStyleSheetForBucket(
```

Comment:
Can we cover this by UTs?

Reply:
- style elements are added
- added in correct order
- added only once

Up to you